### PR TITLE
Make it possible to allow collaborator admin access in pet clusters

### PR DESCRIPTION
**Pull request by aermakov-zalando**
Currently we only allow access to e2e clusters, but pet clusters should also work.

### CODE REVIEW

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -271,6 +271,13 @@ custom_dns_zone_nameservers: "" # space seperated list of nameserver IP addresse
 # prefix prepended to ownership TXT records for external-dns
 external_dns_ownership_prefix: ""
 
+# special roles for test/pet clusters
+{{if eq .Cluster.Environment "e2e"}}
+collaborator_administrator_access: "false"
+{{else}}
+collaborator_administrator_access: "true"
+{{end}}
```
Comment (mikkeloscar) [inline cluster/config-defaults.yaml:279]: Is the logic not reversed here? Should be `true` in case of e2e?

Comment (aermakov-zalando) [reply cluster/config-defaults.yaml:279]: Oops.

**Comment by mikkeloscar** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**